onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -832,13 +832,13 @@ def _where_input_wrangler(
         matcher=lambda sample: len(sample.args) > 0,
         reason="this ATen overload only supports one tensor as input by design",
     ),
-    skip(
+    xfail(
         "min_other",  # aten_min_other(self, other)
         matcher=lambda sample: len(sample.args) == 0
         or (len(sample.args) > 0 and isinstance(sample.args[0], int)),
         reason="this ATen overload only support one tensor as input and another tensor as args",
     ),
-    skip(
+    xfail(
```

Comment:
There is one thing I would like to make sure: when xfail is properly raised by pytest, do the rest of the subtests continue to run as they should?

To verify, maybe just print a simple count for one of the tests and make sure the numbers are equal. Pytest's report is not reliable because it doesn't count subtests when tests succeed. It only counts the bigger tests.

Reply:
When it's skip:

```bash
======================================================================= short test summary info ========================================================================
SKIPPED [1] ../anaconda3/envs/pytorch-dev/lib/python3.9/unittest/case.py:117: Function checker is not available before ONNX 1.14
SKIPPED [8] onnxscript/tests/function_libs/torch_lib/ops_test_common.py:467: this Aten overload only support one tensor as input and {dim,keepdim} as kwargs by design
============================================================ 2 passed, 9 skipped, 625 deselected in 13.78s =============================================================
```

When it's xfail:

```bash
======================================================================= short test summary info ========================================================================
SKIPPED [1] ../anaconda3/envs/pytorch-dev/lib/python3.9/unittest/case.py:117: Function checker is not available before ONNX 1.14
XFAIL onnxscript/tests/function_libs/torch_lib/ops_test.py::TestOutputConsistencyEagerCPU::test_output_match_opinfo__all_dim_cpu_float32
  reason: this Aten overload only support one tensor as input and {dim,keepdim} as kwargs by design
XFAIL onnxscript/tests/function_libs/torch_lib/ops_test.py::TestOutputConsistencyEagerCPU::test_output_match_opinfo__all_dim_cpu_float32
  reason: this Aten overload only support one tensor as input and {dim,keepdim} as kwargs by design
XFAIL onnxscript/tests/function_libs/torch_lib/ops_test.py::TestOutputConsistencyEagerCPU::test_output_match_opinfo__all_dim_cpu_float32
  reason: this Aten overload only support one tensor as input and {dim,keepdim} as kwargs by design
XFAIL onnxscript/tests/function_libs/torch_lib/ops_test.py::TestOutputConsistencyEagerCPU::test_output_match_opinfo__all_dim_cpu_float32
  reason: this Aten overload only support one tensor as input and {dim,keepdim} as kwargs by design
XFAIL onnxscript/tests/function_libs/torch_lib/ops_test.py::TestOutputConsistencyFullGraphCPU::test_output_match_opinfo__all_dim_cpu_float32
  reason: this Aten overload only support one tensor as input and {dim,keepdim} as kwargs by design
XFAIL onnxscript/tests/function_libs/torch_lib/ops_test.py::TestOutputConsistencyFullGraphCPU::test_output_match_opinfo__all_dim_cpu_float32
  reason: this Aten overload only support one tensor as input and {dim,keepdim} as kwargs by design
XFAIL onnxscript/tests/function_libs/torch_lib/ops_test.py::TestOutputConsistencyFullGraphCPU::test_output_match_opinfo__all_dim_cpu_float32
  reason: this Aten overload only support one tensor as input and {dim,keepdim} as kwargs by design
XFAIL onnxscript/tests/function_libs/torch_lib/ops_test.py::TestOutputConsistencyFullGraphCPU::test_output_match_opinfo__all_dim_cpu_float32
  reason: this Aten overload only support one tensor as input and {dim,keepdim} as kwargs by design
======================================================= 2 passed, 1 skipped, 625 deselected, 8 xfailed in 14.02s =======================================================
```

It seems to work. LMK if there is anything weird.

Reply:
I guess this approach actually counts subTests. Not sure if it's a pros or cons though.

Reply:
Thanks! This looks good - the number of subfails matches the number of skips or xfails below.